### PR TITLE
Rm globals ##cons

### PR DESCRIPTION
**Pull request by luc-tielen**
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

Removes some globals from cons (input.c).

### CODE REVIEW

**Comment by trufae** [comment]
The asan fail is another uaf i found yesterday. Unrelated to this pr

File: libr/cons/more.c [hunk 1]
```diff
@@ -15,6 +15,7 @@ static const char *r_cons_more_help = \
 	" ?        - show this help\n"
 	"\n";
 
+// XXX dead code?
```
Comment (trufae) [inline libr/cons/more.c:18]: If 0 them if not used

**Comment by trufae** [comment]
634
more.c:8:20: error: ‘r_cons_more_help’ defined but not used [-Werror=unused-variable]
[635](https://github.com/radareorg/radare2/actions/runs/5244775068/jobs/9471255473?pr=21885#step:3:636)
    8 | static const char *r_cons_more_help = \
[636](https://github.com/radareorg/radare2/actions/runs/5244775068/jobs/9471255473?pr=21885#step:3:637)
      |                    ^~~~~~~~~~~~~~~~
[637](https://github.com/radareorg/radare2/actions/runs/5244775068/jobs/9471255473?pr=21885#step:3:638)
cc1: all warnings being treated as errors

**Comment by luc-tielen** [comment]
So I fixed the warning, but now the whole file is commented out.. maybe we can just remove it @trufae ?

**Pull request merged**